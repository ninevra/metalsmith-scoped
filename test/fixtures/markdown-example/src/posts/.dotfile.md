---
description: a dotfile
---
This'll only get converted to markdown *if* `{dot: true}` is passed to `marked`.
