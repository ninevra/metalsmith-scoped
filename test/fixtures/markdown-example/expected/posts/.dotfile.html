<p>This&#39;ll only get converted to markdown <em>if</em> <code>{dot: true}</code> is passed to <code>marked</code>.</p>
